### PR TITLE
add readme file

### LINKED ISSUE #1: Add a README file
Create README.md file to give an overview of what this repo is for.

### PR DESCRIPTION
**Pull request by realalysonla**
Fixes #1 

Added a readme file so people will know what this readme is about. :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Treehouse
+
+This repo is for all my course file completed through teamtreehouse.com
```
Comment (realalysonla) [inline README.md]: Maybe add more info & a picture?

**Comment by MichaelPay** [comment]
will *know* what this readme is about. :+1:

**Comment by cameronmosser** [comment]
@realalysonla great videos, you're one of the best mentors on Treehouse! To piggyback on what @MichaelPay said, I too noticed the same spelling error and wanted to get some practice in the comments section by letting you know. Keep up the fantastic work!

**Comment by realalysonla** [comment]
Ah thanks! @cameronmosser (and @MichaelPay). Fixed 😊

**Comment by cameronmosser** [comment]
Hi Alyson,

I couldn’t find a DM feature on GitHub so hopefully replying to this email notifies you.

I’m currently a student at Code Louisville here in Louisville, KY and I was wondering if you would by chance consider being my mentor.
You’re concise and easy to understand and it would mean the world to me if you could spend just an hour taking me through the core competencies of JS.

If not, I totally understand, I’m sure you’re a very busy bee!

Cheers!

-Cameron M.

**Comment by alysonla** [comment]
@cameronmosser awesome, that you are learning to code JS! I'm flattered that you would ask, however, as a data scientist, I actually don't know much (any?) JS at all. However, also feel free to keep in touch via twitter - https://twitter.com/alysonlaaa. Best of luck in your course. 📚 💯

**Pull request merged**